table_evaluator/table_evaluator.py

```diff
@@ -46,6 +43,7 @@ def __init__(self, real: pd.DataFrame, fake: pd.DataFrame, cat_cols=None, unique
         self.fake = fake.copy()
         self.comparison_metric = getattr(stats, metric)
         self.verbose = verbose
+        self.random_seed = seed
```

Comment:
I'd add the below mentioned seed setting here.

Reply:
It had to be placed specifically before those train_test_split functions in order to produce reproducible results.